LogicalTypes.md

```diff
@@ -130,8 +130,12 @@ milliseconds from the Unix epoch, 00:00:00.000 on 1 January 1970, UTC.
 microsecond precision. It must annotate an `int64` that stores the number of
 microseconds from the Unix epoch, 00:00:00.000000 on 1 January 1970, UTC.
 
-### INTERVAL
+### INTERVAL types
 
+#### INTERVAL
+`INTERVAL` is deprecated. Please use the more precise `INTERVAL_YEAR_MONTH` and
+`INTERVAL_DAY_TIME` per SQL spec.
```

Comment:
Sorry I meant if we could add a link to the sql spec in the md file, it might be useful for folks reading in the future.